metro-ai-suite/smart-intersection/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: LicenseRef-Intel-Edge-Software
+# This file is licensed under the Limited Edge Software Distribution License Agreement.
+
+import pytest
+import time
+from tests.utils.utils import run_command
+
+DOCKER_COMPOSE_FILE = "compose.yml" 
+SMART_INTERSECTION_URL = "https://localhost"
+GRAFANA_URL = "http://localhost:3000"
+INFLUX_DB_URL = "http://localhost:8086"
+NODE_RED_URL = "http://localhost:1880"
```

Comment:
Consider using dotenv or similar to get this values.
That way, you can use the test to verify a remote host as well, e.g. for helm deployment in remote k8s cluster
(You could also port forward those into localhost of the test machine and use it like that)